src/util/util.js

```diff
@@ -385,6 +385,28 @@ export function isClosedPolygon(points: Array<Point>): boolean {
     return Math.abs(calculateSignedArea(points)) > 0.01;
 }
 
+/**
+ * Retruns the maximum value of an element in an array,
```

Comment:
Typo `Retruns` and on the second line `retured`. You should also mark this function as `@private` in the JSDoc header